spooky-oysters/bangazon-web-app
# Adding product to order

## Link to Ticket
#13 User Can View Product Detail

## Description of Proposed Changes

- When viewing the product detail, we now have an 'Add to Cart' button

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout km-product-to-order
```
1. Navigate to an item list (search for an item)
1. Select details
1. Click the add to order should be directed to order with product added
1. Close the order
1. Add another item, should be directed to a new order with that product on it

## Impacted Areas in Application

List general components of the application that this PR will affect:

* 

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
